template-only-test/template_infra_test.go

```diff
@@ -67,7 +67,7 @@ func SetUpAccount(t *testing.T) {
 	fmt.Println("::group::Setting up account")
 	shell.RunCommand(t, shell.Command{
 		Command:    "make",
-		Args:       []string{"infra-set-up-account", "ACCOUNT_NAME=prod"},
+		Args:       []string{"infra-set-up-account", "ACCOUNT_NAME=dev"},
```

Comment:
This change is intentional, right?

Reply:
Yeah, for testing the Template CI was creating the prod account, but the environments are configured such that the build repository is in the dev account i.e. `shared_network_name = "dev"` in app-config module. So for the CI I think it makes sense to just create everything in the dev account. In the future we could potentially expand CI to run across accounts.